packages/lesswrong/components/common/TabNavigationMenu/SubforumsList.tsx

```diff
@@ -40,10 +52,12 @@ const SubforumsList = ({ onClick, classes }) => {
   
   const { TabNavigationSubItem } = Components
 
+  
   return (
     <span>
       <AnalyticsContext pageSubSectionContext="menuSubforumsList">
         <div>
+          <div className={classes.title}>Subforums</div>
```

Comment:
Using "Subforums" as a heading rather than having it after every subforum name to make them more likely to fit on one line